### DISCUSSION TITLE
Disable mob backpack spawns?

### DISCUSSION THREAD

**Discussion by WillGriff64** [P3pp3rF1y/SophisticatedBackpacks#818] [Q&A]
Is there a way to disable the mobs that spawn with backpacks equipped? I cant find a gamerule or command to achieve this. Thanks!

**Comment by P3pp3rF1y** [comment]
Just set the chance of these spawning to 0 in config and it's done

**Reply by kostlavyjdamaka** [reply]
If it's not too much trouble, can you tell me where to do it? I can't find it

**Reply by P3pp3rF1y** [reply]
just search for chance in sophisticatedbackpacks-server.toml and change that. Also make sure you're changing this while the game is running if you're playing pre 1.21 or before you load your world in 1.21+.
